File: back/src/LaMaps.Desktop/Gpx/GpxDto.cs
using System.Collections.Generic;
using NetTopologySuite.IO;

namespace LaMaps.Desktop.Controllers
{
    public class GpxDto
    {
        public List<GpxRoute> Routes { get; set; }
        public List<GpxWaypoint> Waypoints { get; set; }
        public List<Track> Tracks { get; set; }
        public LaMetadata Metadata { get; set; }
    }
}

File: front/src/api/models/GpxDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { GpxRoute } from './GpxRoute';
import type { GpxWaypoint } from './GpxWaypoint';
import type { LaMetadata } from './LaMetadata';
import type { Track } from './Track';

export type GpxDto = {
    routes?: Array<GpxRoute> | null;
    waypoints?: Array<GpxWaypoint> | null;
    tracks?: Array<Track> | null;
    metadata?: LaMetadata;
}